utils/build-icons/lib/config.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
         removeDimensions: true
       },
       {
-        removeAttrs: { attrs: '(stroke|fill)' }
+        removeAttrs: false
       }
     ]
   }
```

Comment:
Why are we removing so much of the optimization? I know @aselbie mentioned he's seen it fail or mess with the intended vectors (definitely curious to hear a little more about that).

In the last SVG icon set I helped implement I believe we tuned it pretty similarly to how this had been configured and only whitelisted a couple attributes (`class` and `currentColor`?) used in targeting individual SVG nodes with CSS.